calibration/CTDMOG/CGINS-CTDMOG-13421__20150510.csv

```diff
@@ -10,16 +10,16 @@ serial,name,value,notes
 37-13421,CC_i,1.278334e-05,
 37-13421,CC_j,1.742975e-05,
 37-13421,CC_p_range,1450,
-37-13421,CC_pa0,0.1941297,
-37-13421,CC_pa1,0.004950054,
-37-13421,CC_pa2,-2.066723e-11,
-37-13421,CC_ptca0,525111.2,
-37-13421,CC_ptca1,2.700952,
-37-13421,CC_ptca2,-0.04099572,
-37-13421,CC_ptcb0,25.48337,
-37-13421,CC_ptcb1,0.000275,
+37-13421,CC_pa0,1.648240e+00,
+37-13421,CC_pa1,4.587154e-03,
+37-13421,CC_pa2,-2.710773e-11,
+37-13421,CC_ptca0,5.249761e+05,
+37-13421,CC_ptca1,1.147109e+01,
+37-13421,CC_ptca2,1.649877e-01,
+37-13421,CC_ptcb0,2.499737e+01,
+37-13421,CC_ptcb1,7.500000e-05,
 37-13421,CC_ptcb2,0,
-37-13421,CC_ptempa0,-72.90904,
-37-13421,CC_ptempa1,0.05137249,
-37-13421,CC_ptempa2,-7.216569e-07,
-37-13421,CC_wbotc,-6.0918e-08,
+37-13421,CC_ptempa0,1.407389e+02,
+37-13421,CC_ptempa1,-5.987758e-02,
+37-13421,CC_ptempa2,-1.034170e-06,
+37-13421,CC_wbotc,-6.091844e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT